### PR TITLE
Maint/drop python 3.9

### PR DESCRIPTION
**Pull request by chrpetre**
remove support for Python 3.9, update PyAnsys dependencies version

### CODE REVIEW

File: poetry.lock
Comment (lboucin) [inline poetry.lock:1]: @chrpetre There are a lot of changes in your poetry.lock. @Revathyvenugopal162 put the poetry.lock file in the .gitignore file. So, I'm not sure how we should maintain this file. Revathy could you please give us advice on the way to maintain the poetry.lock file? Thanks once more in advance for your great support!

**Review by lboucin — COMMENTED** [review]
LGTM apart from the poetry.lock file that I'm not sure what is the right way to maintain it and if we should modify it (for me the .lock file may vary from one dev env to another and should not be modified except for intentional maintenance purpose). Let's wait for Revathy advice before merging this one.

**Review by lboucin — APPROVED** [review]
Actually, Revathy removed the poetry.lock file from the .gitgnore file. So, I guess we're good to go this way.

**Review by Revathyvenugopal162 — APPROVED** [review]

**Comment by Revathyvenugopal162** [comment]
we need to keep the poetry lock file.

**Pull request merged**